rest/src/main/kotlin/builder/interaction/ApplicationCommandStateHolder.kt

```diff
@@ -17,6 +18,11 @@ internal class ApplicationCommandModifyStateHolder {
 
     var options: Optional<MutableList<OptionsBuilder>> = Optional.Missing()
 
+    var defaultMemberPermissions: Optional<Permissions?> = Optional.Missing()
+    var dmPermission: OptionalBoolean? = OptionalBoolean.Missing
+
+
+    @Deprecated("danger default_permission will soon be deprecated. You can instead set default_member_permissions to \"0\" to disable the command by default and/or set dm_permission to false to disable globally-scoped commands inside of DMs with your app")
```

Comment:
Probably wanna set this to error level

Reply:
Not if the API still accepts it

Reply:
The fact that *it should be changed asap* implies that it should be marked as error to force people to move away from it

Reply:
If it still works with the Discord API I am not convinced that introducing a compiler error is the right thing to do

Reply:
I think you are right